salt/kube-proxy/init.sls

```diff
@@ -0,0 +1,24 @@
+include:
+  - repositories
+
+conntrack-tools:
+  pkg.installed
+
+kube-proxy:
+  pkg.installed:
+    - pkgs:
+      - iptables
+      - conntrack-tools
```

Comment:
It looks like we're installing conntrack-tools twice, it's probably best to remove the one at line 4?